src/shogun/regression/LeastAngleRegression.h

```diff
@@ -184,20 +188,49 @@ class CLeastAngleRegression: public CLinearMachine
 	virtual const char* get_name() const { return "LeastAngleRegression"; }
 
 protected:
-	virtual bool train_machine(CFeatures* data=NULL);
+	/**
+	* An interface method to - this is called by the superclass's (CLinearMachine)
+	* train() method.  This method checks to see if data is a dense feature vector,
```

Comment:
pls doxygen explicitly reference methods you refer to, so that doxygen generates links in the html ouput